ironfish-cli/src/utils/s3.ts

```diff
@@ -309,6 +318,31 @@ export function getS3Client(
   })
 }
 
+export function getR2S3Client(credentials: {
+  r2AccessKeyId: string
+  r2SecretAccessKey: string
+}): S3Client {
+  return new S3Client({
+    region: 'auto',
+    endpoint: R2_ENDPOINT,
+    credentials: {
+      accessKeyId: credentials.r2AccessKeyId,
+      secretAccessKey: credentials.r2SecretAccessKey,
+    },
+  })
+}
+
+export async function getR2Credentials(): Promise<R2Secret | undefined> {
+  const client = new SecretsManagerClient({ region: 'us-east-1' })
```

Comment:
I am not sure about this region config. If the ceremony server has terraform config, it prob wont need it. 
Maybe try it without region on the github workflow